go/libkb/secret_store.go

```diff
@@ -64,9 +92,13 @@ func GetConfiguredAccounts(c SecretStoreContext) ([]keybase1.ConfiguredAccount,
 }
 
 func ClearStoredSecret(g *GlobalContext, username NormalizedUsername) error {
-	secretStore := NewSecretStore(g, username)
-	if secretStore == nil {
+	if g.SecretStoreAll == nil {
 		return nil
 	}
-	return secretStore.ClearSecret()
+	return g.SecretStoreAll.ClearSecret(username)
+}
+func NewTestSecretStoreAll(c SecretStoreContext, g *GlobalContext) SecretStoreAll {
```

Comment:
I have been going around in circles on this. It must be accessible from test_common.go, I believe mainly so tests in other packages can use it, so it can't be compiled test-only. I will try solving the osx breakage by putting a no-op NewTestSecretStoreAll() in the darwin file, same on Android.